prefs.js

```diff
@@ -14,7 +14,11 @@ const ExtensionUtils = imports.misc.extensionUtils;
 const WORKSPACE_COUNT_KEY = 'workspace-count';
 const WORKSPACE_INDEX = 'workspace-index';
 const WALLPAPERS_KEY = 'workspace-wallpapers';
-const CURRENT_WALLPAPER_KEY = 'picture-uri';
+let   CURRENT_WALLPAPER_KEY = 'picture-uri';
```

Comment:
Please change to const to be consistent with other variables.

Reply:
CURRENT_WALLPAPER_KEY can't be a constant now, it need to be variable because it has to change based on the gnome scheme color (DARK or LIGHT)